scripts/fix-browser-order.js

```diff
@@ -1,28 +1,29 @@
-#!/usr/bin/env node
```

Comment:
This&nbsp;is&nbsp;correct for&nbsp;this&nbsp;file.

```suggestion
#!/usr/bin/env node
```